### PR TITLE
Minor edit to fhirAuthorization field documentation and example

### PR DESCRIPTION
**Pull request by bakerdb**
Modified example to align with parameters table by adding "fhirAuthorization" and "encounter" fields."  
Reworded description of "fhirAuthorization" parameter.
This change is responsive to Risk R7.

### CODE REVIEW

File: docs/specification/1.0.md [hunk 1]
```diff
@@ -92,9 +92,11 @@ curl
    "hookInstance" : "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
    "fhirServer" : "http://hooks.smarthealthit.org:9080",
    "hook" : "patient-view",
+   "fhirAuthorization" : "a93aa0b328a;nbeaoajwubelwodkca038",
```
Comment (kpshek) [inline docs/specification/1.0.md]: This should be a object, not a string.

File: docs/specification/1.0.md [hunk 2]
```diff
@@ -92,9 +92,11 @@ curl
...
    "user" : "Practitioner/example",
    "context" : [],
    "patient" : "1288992",
+   "encounter" : "home.id",
```
Comment (kpshek) [inline docs/specification/1.0.md]: Nit: consider making this a number to align with the patient value

File: docs/specification/1.0.md [hunk 3]
```diff
@@ -92,9 +92,11 @@ curl
    "hookInstance" : "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
    "fhirServer" : "http://hooks.smarthealthit.org:9080",
    "hook" : "patient-view",
+   "fhirAuthorization" : "params.tokenUri",
```
Comment (kpshek) [inline docs/specification/1.0.md]: This should be a JSON object like:

```json
"fhirAuthorization" : {
  "access_token" : "some-opaque-fhir-access-token",
  "token_type" : "bearer",
  "expires_in" : 300,
  "scope" : "patient/Patient.read patient/Observation.read",
  "subject" : "the-cds-service-client-identifier"
}
```

File: docs/specification/1.0.md [hunk 4]
```diff
@@ -126,7 +128,7 @@ Field | Description
 `hook` |*string* or *URL*. The hook that triggered this CDS Service call<br />(todo: link to hook documentation)
 <nobr>`hookInstance`</nobr> |*string*.  A UUID for this particular hook call (see more information below)
 `fhirServer` |*URL*.  The base URL EHR's [FHIR](https://www.hl7.org/fhir/) server. The scheme should be `https`
-`fhirAuthorization` | *object*. The OAuth 2 authorization providing access to the EHR's FHIR server. See the [FHIR Resource Access](#fhir-resource-access) heading of the security section for more information.
+`fhirAuthorization` | *object*. The OAuth 2.0 bearer access token granting the CDS Service access to FHIR resources. See the [FHIR Resource Access](#fhir-resource-access) heading of the security section for more information.
```
Comment (kpshek) [inline docs/specification/1.0.md]: The `fhirAuthorization` field isn't the bearer token. It is an object that contains the bearer token along with supplemental information related to said token.

File: docs/specification/1.0.md (hunk 3 above)
Comment (bakerdb) [reply docs/specification/1.0.md]: @kpshek, agreed.  Although [RFC6750](https://tools.ietf.org/html/rfc6750) discusses "bearer tokens" (using lower case), example code consistently uses upper case.  Thanks for noticing!

File: docs/specification/1.0.md (hunk 4 above)
Comment (bakerdb) [reply docs/specification/1.0.md]: @kpshek.  Agree that fhirAuthorization contains more than the access token.   I think the most accurate description is to describe it as an "authorization structure."

**Pull request merged**